proto/quicksilver/airdrop/v1/messages.proto

```diff
@@ -8,4 +8,26 @@ import "google/api/annotations.proto";
 option go_package = "github.com/ingenuity-build/quicksilver/x/airdrop/types";
 
 // Msg defines the airdrop Msg service.
-service Msg {}
+service Msg {
+    rpc Claim(MsgClaim) returns(MsgClaimResponse) {
+        option (google.api.http) = {
+            post : "/quicksilver/tx/v1/airdrop/claim"
+            body : "*"
+        };
+    }
+}
+
+message MsgClaim {
+    option (gogoproto.equal) = false;
+    option (gogoproto.goproto_getters) = false;
+
+    string chain_id = 1 [ (gogoproto.moretags) = "yaml:\"chain_id\"" ];
+    int32 action = 2 [ (gogoproto.moretags) = "yaml:\"action\"" ];
+    string address = 3 [ (gogoproto.moretags) = "yaml:\"address\"" ];
+}
```

Comment:
This needs to be able to take some arbitrary []bytes because the payload that needs to be verified will differ between action types (or nil for local-chain claims).

Reply:
For example; the osmosis lockup proofs look like this:

https://gist.github.com/joe-bowman/e1eff2fcdf68d6ef5b9ffcf89a04d7d8

So we can either make the payload one of OsmosisPayload|CrescentPayload|... or free []bytes that we validate later.

This latter suggestion will work better when it comes to wasm based validations, as we won't _want_ to define data structures in the module then.

Reply:
I favor the []byte route... more flexible.